doc/classes/EditorImportPlugin.xml

```diff
@@ -1,8 +1,53 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="EditorImportPlugin" inherits="Reference" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		Used by the editor to import resources from files.
 	</brief_description>
 	<description>
+		EditorImportPlugins provide a way to extend the editor's resource import functionality. They can be used to import resources from custom files or to provide alternatives to the editor's existing importers. EditorImportPlugins must be registered by an [EditorPlugin] by using [method EditorPlugin.add_import_plugin].
```

Comment:
Avoid passive voice
`They can be used` -> `Use them to`
Use the direct voie
`EditorImportPlugins must be registered by an [EditorPlugin] by using [method EditorPlugin.add_import_plugin]` -> `Register your EditorImportPlugin with [method EditorPlugin.add_import_plugin]`